pydocx/export/__init__.py

```diff
@@ -4,10 +4,11 @@
     unicode_literals,
 )
 
-from pydocx.export.html import PyDocXHTMLExporter
+from pydocx.export.html import PyDocXHTMLExporter, PyDocXHTMLExporterWithImageResize
```

Comment:
The mixin should be an optional include, not included in the standard library by default.